sechub-integrationtest/src/main/java/com/daimler/sechub/integrationtest/internal/MockEmailAccess.java

```diff
@@ -86,7 +101,7 @@ public MockEmailEntry findMailOrFail(String email, String subject, int maxSecond
 			sb.append("Did not found mail containing:\n-emailadress (TO/CC/BCC): ");
 			sb.append(email);
 			sb.append("\n-subject: '");
-			sb.append(subject);
+			sb.append(subjectRegexp);
```

Comment:
Hmm.. we use this variable for regular expressions but also for normal string search - when reading this I was thinking the parameter would be only for regular expression. Maybe we should still use "subject" or "subjectContent" instead?

Reply:
I this case, it is really the regexp used for searching the subject.
Will improve the message \n-subject:... so it's more clear.